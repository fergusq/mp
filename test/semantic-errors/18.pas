program p;
begin var a:integer;
writeln(a[0])end.
