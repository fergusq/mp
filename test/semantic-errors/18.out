Semantic error: Type mismatch: a is indexed but it is not an array
